### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by tnyandoro**
i have read and used all the details of the Bookstore design in Zeplin.
please review my work
Login to Zeplin (hint: you will find the credentials in the sneak peek of the project).
Open the project Bookstore CMS.
Use the information detailed in Zeplin to style the website.
Your final design should match Zeplin's design.

### CODE REVIEW

File: src/components/BookProgress.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto&display=swap');
 
 .book-progress {
   display: flex;
```
Comment (oxenprogrammer) [inline src/components/BookProgress.css:4]: **Changes Required**
Good effort so far, bravo. However;
- [x] The CSS selector property `display` with the value of `flex` is repeated multiple times. You want to follow best CSS practice of **Don't Repeat Yourself. (DRY).** Please let us refactor it by creating for instance a `.display-flex` in your `.jsx` file and having a `.display-flex { display: flex}` in your CSS file. Please do this for all repeated Selector properties i.e `(color, font-size, etc)` in all your CSS files.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @tnyandoro , 

Good job so far!
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_
**Optional Suggestion**
- [x] Apparently the app overflows horizontally and it's too large for a normal laptop screen. Please let us add `overflow-x: hidden;` CSS selector property to the body or main div to fix this.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/BookProgress.css (hunk 1 above)
Comment (tnyandoro) [reply src/components/BookProgress.css:4]: Overflow hidden added to cater for small laptop screens 
Display flex repeats Sorted

**Comment by tnyandoro** [comment]
in the books .css file resing properties is conflicting with the style lint errors. like puting the .list class twice or book infor so i won't attempt

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @tnyandoro, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**